tests/support_server/setup.pm

```diff
@@ -283,6 +280,98 @@ sub setup_xdmcp_server {
     $xdmcp_server_set = 1;
 }
 
+sub setup_iscsi_server {
+    return if $iscsi_server_set;
+
+    # If no LUN number is specified we must die!
+    my $num_luns = get_required_var('NUMLUNS');
+
+    # A second disk for iSCSI LUN is mandatory
+    my $hdd_lun_size = get_required_var('HDDSIZEGB_2');
+
+    # Integer part of the LUN size is keep and can't be lesser than 1GB
+    my $lun_size = int($hdd_lun_size / $num_luns);
+    die "iSCSI LUN cannot be lesser than 1GB!" if ($lun_size < 1);
+
+    # Are we using virtio or virtio-scsi?
+    # Note: it should be possible to do this in pure Perl code...
+    my $hdd_lun = script_output "
```

Comment:
Yes, you're right!
And the 'cut' is not needed also: I just need to remove the "/dev/" in front of $hdd_lun when I use it.